defi/src/storeGetProtocols.ts

```diff
@@ -175,6 +175,7 @@ export async function storeGetProtocols({
       mcap: protocol.mcap,
       gecko_id: protocol.gecko_id,
       parent: protocol.parentProtocol,
+      ...(protocol.deprecated || protocol.name.includes("eprecated") ? {deprecated: true} : {})
```

Comment:
why check protocol name?

Reply:
some protocols like rage trade v1 have deprecated in their name https://api.llama.fi/lite/protocols2